i18n/fr.yaml

```diff
@@ -0,0 +1,77 @@
+404_title: '404'
+404_description: "Oops! Cette page n'existe pas!"
+subheader: ''
+
+# menu ----
+home: "Home"
+about-us: 'About'
+mission: "Mission"
+get-involved: 'Get involved'
+global-team: 'Meet the global team'
+history: 'History'
+news: 'News'
+directory: 'Directory'
+activities: 'Activities'
+events: 'Events'
+mentoring: 'Mentoring Program'
+coordinated: 'Coordinated by the Global Team'
```

Comment:
This is a great comment! Yes its on purpose, RIght now, its indicating what needs translating. Since my French is very poor, I dont dare do it. So I leave it in English, so that when the translation team starts their work, its easy to see what the text is in English, and place the appropriate translation.